src/components/layout/jump-indices/nav.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import { localize } from 'components/localization'
+import LogoCombinedShape from 'images/svg/logo-combined-shape.svg'
+import { QueryImage } from 'components/elements'
+import { Container, Flex, SectionContainer } from 'components/containers'
+import device from 'themes/device.js'
+
+export const Section = styled(SectionContainer)`
+    background-color: var(--color-black);
+    width: 100%;
+    position: fixed;
+    z-index: 3;
+    top: 0;
+    padding: 22px 1%;
+    height: 72px;
+
+    @media ${device.tabletL} {
+        padding: 16px;
+        height: 48px;
+    }
+
+    @media ${device.mobileL} {
+        padding: 16px 0;
+    }
+`
+
+const ContentContainer = styled(Container)`
+    align-items: center;
+    justify-content: space-between;
+
+    @media ${device.tabletL} {
+        justify-content: center;
+    }
+`
+
+const LogoWrapper = styled(Flex)`
+    flex-direction: row;
```

Comment:
All the `Flex` components default to row as the flex direction. We can omit this from all the flex extensions.

Reply:
Fixed in latest commit.